packages/cli/src/commands/cms.ts

```diff
@@ -0,0 +1,62 @@
+import { GluegunToolbox } from "gluegun";
+import { join } from "path";
+
+module.exports = {
+  name: "cms",
+  alias: ["cms"],
+  hidden: true,
+  run: async (toolbox: GluegunToolbox) => {
+    const mainCmsPath = join(toolbox.defaultThemeLocation, "cms");
+
+    const swCmsPath = join(`.shopware-pwa`, `sw-cms`);
+    const swPluginsPath = join(`.shopware-pwa`, `pwa-bundles-assets`);
+
+    // Aliases and componentsMap to save
+    const aliases = {};
+    const cmsComponentsMap = {
+      sections: {},
+      blocks: {},
+      elements: {},
+    };
+
+    // Read theme cms settings
+    await toolbox.resolveCms(mainCmsPath, aliases, cmsComponentsMap);
+
+    // Override CMS by plugins
+    const pluginsConfig = await toolbox.filesystem.readAsync(
+      join(`.shopware-pwa`, "pwa-bundles.json"),
```

Comment:
could we decide which format of string literals is preferred in the entire project?
We've got inconsistent strategy to deal with strings;) 
sometimes its "" (double quoted), sometimes ' ' (single quoted), and sometimes it's a new  ` ` (template string). 
Even though the template strings are desired way to deal with dynamic parts and string concatonations, we still don't know if double of single quoted string is preferred when it comes to deal with simple strings ;) (the same thing with module imports). Please choose a good strategy, I believe it could be spreaded over entire project.

Reply:
yeah, we should configure prettier for that :)
* template strings - strings with dynamic parts
* double-quote otherwise

currently, double quotes are set for example in ts files, but not in .vue files.
As an action point, I'm creating an issue

Reply:
https://github.com/DivanteLtd/shopware-pwa/issues/634